### PR TITLE
fix : enum has null value, it will be shown

### LINKED ISSUE #9682: when enum has null, swagger ui doesn't show "null"
### Q&A (please complete the following information)
 - OS: [macOS]
 - Browser: [chrome]
 - Version: [122]
 - Method of installation: [npm]
 - Swagger-UI version: [5.1.10]
 - Swagger/OpenAPI version: [OpenAPI 3.0]

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
openapi: 3.0.0
info:
  title: Sample API
  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
  version: 0.1.9
servers:
  - url: http://api.example.com/v1
    description: Optional server description, e.g. Main (production) server
  - url: http://staging-api.example.com
    description: Optional server description, e.g. Internal staging server for testing
paths:
  /users:
    get:
      summary: Returns a list of users.
      description: Optional extended description in CommonMark or HTML.
      parameters:
        - in: query
          name: sort
          description: Sort order
          schema:
            type: string
            nullable: true
            enum:
              - asc
              - desc
              - null 
      responses:
        '200':
          description: A JSON array of user names
          content:
            application/json:
              schema: 
                type: string
                nullable: true
                enum:
                  - asc
                  - desc
                  - null 
```

Swagger-UI configuration options:
```js
SwaggerUI({
  // no custom option
})
```

### Describe the bug you're encountering

Both Parameters and Responses section,
when there are enum type variables and the enum variables have null value, null value is not shown.

### To reproduce...

use above yaml 

### Expected behavior

null value should be shown

### Screenshots

스크린샷 2024-03-12 오후 12 44 04

### PR DESCRIPTION
**Pull request by bejewel-kyoungmin**
### Description

```yaml
openapi: 3.0.0
info:
  title: Sample API
  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
  version: 0.1.9
servers:
  - url: http://api.example.com/v1
    description: Optional server description, e.g. Main (production) server
  - url: http://staging-api.example.com
    description: Optional server description, e.g. Internal staging server for testing
paths:
  /users:
    get:
      summary: Returns a list of users.
      description: Optional extended description in CommonMark or HTML.
      parameters:
        - in: query
          name: sort
          description: Sort order
          schema:
            type: string
            nullable: true
            enum:
              - asc
              - desc
              - null 
      responses:
        '200':
          description: A JSON array of user names
          content:
            application/json:
              schema: 
                type: string
                nullable: true
                enum:
                  - asc
                  - desc
                  - null 
```

in this case, in parameters, and response has enum type schema, and enum has null value.
but swagger ui doesn't show null value. it makes reader confused.

### Motivation and Context

Fixes #9682 

### How Has This Been Tested?

I have built swagger ui with fixed version

### Screenshots (if appropriate):

before | after

<table>
<tr>
<td>before</td>
<td>after</td>
</tr>
<tr>
<td>
스크린샷 2024-03-12 오후 12 44 04

</td>
<td>

스크린샷 2024-03-12 오후 1 10 27

</td>
</tr>
</table>

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/core/components/enum-model.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const EnumModel = ({ value, getComponent }) => {
   return <span className="prop-enum">
     Enum:<br />
     <ModelCollapse collapsedContent={ collapsedContent }>
-      [ { value.join(", ") } ]
+      [ { value.map((v) => v === null ? "null" : v).join(", ") } ]
```
Comment (char0n) [inline src/core/components/enum-model.jsx:10]:
```suggestion
      [ { value.map(String).join(", ") } ]
```

File: src/core/components/parameter-row.jsx [hunk 2]
```diff
@@ -295,7 +295,7 @@ export default class ParameterRow extends Component {
             <Markdown className="parameter__enum" source={
                 "<i>Available values</i> : " + paramEnum.map(function(item) {
                     return item
-                  }).toArray().join(", ")}/>
+                  }).toArray().map((v) => v === null ? "null" : v).join(", ")}/>
```
Comment (char0n) [inline src/core/components/parameter-row.jsx:298]:
```suggestion
                  }).toArray().map(String).join(", ")}/>
```

File: src/core/components/enum-model.jsx (hunk 1 above)
Comment (bejewel-kyoungmin) [reply src/core/components/enum-model.jsx]: cool

**Review by glowcloud — APPROVED** [review]
Thanks for the changes!

I can confirm that rendering for 2020-12 works fine:
Screenshot 2024-05-07 at 12 13 45
so LGTM!

**Review by char0n — APPROVED** [review]

**Pull request merged**